src/main/java/net/sf/jabref/logic/cleanup/CleanupPreset.java

```diff
@@ -0,0 +1,210 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+package net.sf.jabref.logic.cleanup;
+
+public class CleanupPreset {
```

Comment:
Ah ok. 

I would not add the flexibility before it is actually used. I would only introduce something like this if the current code does not handle the situation perfect anymore. Do it as simple as possible until you feel the pain - and only then change it. 

To me, it feels more complex to manage a class with state (multiple booleans) in addition to the preferences, which have to be kept in sync. The JabRefPreferences class must also know this CleanupPreset, making the JabRefPreferences class also more complicated to understand as we would introduce another dependency. And the JabRefPreferences class should be kept as simple as possible as it is so core to JabRef - it is used by almost the whole JabRef code, and this code now all depend on the CleanupPreset as well. 

My suggestion is, use a simple wrapper class and when we want to store these values also for onSave and onOpen, we can simply use different versions of the wrapper class.